pennylane_lightning/lightning_kokkos/_state_vector.py

```diff
@@ -57,33 +57,30 @@ class LightningKokkosStateVector:  # pylint: disable=too-few-public-methods
 
     """
 
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         num_wires,
         dtype=np.complex128,
         device_name="lightning.kokkos",
         kokkos_args=None,
         sync=True,
-    ):  # pylint: disable=too-many-arguments
-        self._num_wires = num_wires
-        self._wires = Wires(range(num_wires))
-        self._dtype = dtype
-
-        self._kokkos_config = {}
-        self._sync = sync
-
-        if dtype not in [np.complex64, np.complex128]:
-            raise TypeError(f"Unsupported complex type: {dtype}")
+    ):
 
         if device_name != "lightning.kokkos":
             raise DeviceError(f'The device name "{device_name}" is not a valid option.')
 
+        super().__init__(num_wires, dtype)
+
         self._device_name = device_name
 
+        self._kokkos_config = {}
+        self._sync = sync
+
+        # Initialize the state vector
         if kokkos_args is None:
-            self._kokkos_state = self._state_dtype()(self.num_wires)
+            self._qubit_state = self._state_dtype()(self.num_wires)
         elif isinstance(kokkos_args, InitializationSettings):
-            self._kokkos_state = self._state_dtype()(self.num_wires, kokkos_args)
+            self._qubit_state = self._state_dtype()(self.num_wires, kokkos_args)
```

Comment:
Rename the state variable to a generic name to be used across the other base classes